README.md

```diff
@@ -21,13 +21,91 @@ Then generate your new project:
 yo node-graphql
 ```
 
+## Generated code
+
+If all default options are entered, the code generated should include a few runnable tests, as well as a runnable server.
+
+### Tests
+
+The generated tests are written using `Jest`, and provide an in memory database using `mongodb-memory-server`, as well as means to test any graphql resolvers and schema validations using `easygraphql-tester`.
+
+The tests can be run as soon as the code is generated with:
+
+```bash
+npm run test
+```
+
+### Source Code
+
+The source code uses `apollo-server-express` along with `express` for graphql and the server, with the `jsonwebtoken` package to handle JWTs. It also uses `lodash`, `merge-graphql-schemas` and `graphql-import-node` for ease of use, as well as `dotenv` for its environment files.
+
+The server can be run with nodemon by:
+
+```bash
+npm run dev
+```
+
+### Database
+
+By default (and currently only choice), the chosen database is noSQL, which uses the `mongoose` package. While the tests can run fine without any setup here, to properly run the server, an instance of `mongoDB` should be created for the system to connect to. Please check the `.env` file and place your database credentials there.
```

Comment:
"By default (and currently only choice), the chosen database is noSQL, which uses the `mongoose` package" => "By default (and currently the only choice) the chosen database is noSQL, which uses the `mongoose` package"

"to properly run the server, an instance of `mongoDB` should be created for the system to connect to." => "to properly run the server an instance of `mongoDB` should be created for the system to connect to."

Reply:
also if people have no idea how to do the above, link to mongoDB install guide?

Reply:
I might put up another task to link all the packages to their git or npm registries and include this in that